front/components/assistant/conversation/input_bar/editor/useCustomEditor.tsx

```diff
@@ -156,6 +157,14 @@ export interface CustomEditorProps {
   disableAutoFocus: boolean;
 }
 
+const CustomBold = Bold.extend({
```

Comment:
Hacker 🙃 !

Reply:
This is the only (and recommended) way to do this on tiptap